src/plugins/global-settings/typography/index.js

```diff
@@ -139,31 +209,162 @@ addFilter( 'stackable.global-settings.inspector', 'stackable/global-typography',
 		model.save()
 	}
 
+	const getIsAllowReset = selector => {
+		const currentFontPair = [ ...FONT_PAIRS, ...customFontPairs ].find( fontPair => fontPair.name === selectedFontPairName )
+		const fontPairStyle = currentFontPair.typography[ selector ]
```

Comment:
I encountered an error here where the `currentFontPair` was undefined and I couldn't anymore access the global settings.

Feels like this is prone now to errors. since I've encountered a few times where the editor stalled and I couldn't access the settings again. Looks like there is an assumption that font pairs are always objects, and if something happened (e.g. user didn't select a font pair, or deleted a font pair), things produce an error. Please adjust so that this case is not assumed